tools/autograd/derivatives.yaml

```diff
@@ -2432,9 +2432,5 @@
   other: non_differentiable
   output_differentiability: [False]
 
-- name: _has_same_storage_numel(Tensor self, Tensor other) -> bool
```

Comment:
The new function still needs an entry here no?

Reply:
The number of differentiable outputs would be zero in this case because the only output is a non-Tensor, so I think that the behavior in terms of codegen would be the same?

Reply:
Ok

Reply:
Here it is!
```
int64_t _storage_numel(c10::DispatchKeySet ks, const at::Tensor & self) {
  auto& self_ = unpack(self, "self", 0);
  #ifndef NDEBUG
  c10::optional<Storage> self__storage_saved =
    self_.has_storage() ? c10::optional<Storage>(self_.storage()) : c10::nullopt;
  c10::intrusive_ptr<TensorImpl> self__impl_saved;
  if (self_.defined()) self__impl_saved = self_.getIntrusivePtr();
  #endif
  auto _tmp = ([&]() {
    at::AutoDispatchBelowADInplaceOrView guard;
    return at::redispatch::_storage_numel(ks & c10::after_autograd_keyset, self_);
  })();
  auto result = _tmp;
  #ifndef NDEBUG
  if (self__storage_saved.has_value())
    AT_ASSERT(self__storage_saved.value().is_alias_of(self_.storage()));
  if (self__impl_saved) AT_ASSERT(self__impl_saved == self_.getIntrusivePtr());
  #endif
  return result;
}
```